tools/restored-receipts-verifier/src/main.rs

```diff
@@ -51,16 +51,7 @@ fn main() -> Result<()> {
         near_config.genesis.config.genesis_height,
         !near_config.client_config.archive,
     );
-    let runtime = NightshadeRuntime::new(
-        home_dir,
-        store,
-        &near_config.genesis,
-        TrackedConfig::from_config(&near_config.client_config),
-        None,
-        near_config.client_config.max_gas_burnt_view,
```

Comment:
Here and eswhere, the sematics changes from `(None, near_config.client_config.max_gas_burnt_view,)` to `(config.client_config.trie_viewer_state_size_limit, config.client_config.max_gas_burnt_view)`. I don't think this should affect anything; I didn't check this very thoroughly; these are auxilary utils, so, even if something breaks, we can fix that.